### ISSUE TITLE
slider (with graph) 

### ISSUE THREAD

**Issue by matthewberryman** [MSFREACH/msf-reach#679]
https://github.com/MSFREACH/msf-reach/issues/527#issuecomment-438382882

I would suggest that we do keep a graph, and make this a graph of number of events per time?

**Comment by LucieGueuning** [comment]
yes, that sounds great! :) I didn't know if that would be feasible though
